### ISSUE TITLE
Standard way to express pledges for conformance resolution

### ISSUE THREAD

**Issue by brennanyoung** [GSA/open-product-accessibility-template#129]
It's not uncommon for VPATs to indicate exceptions and known WCAG violations for a given product. 
This is a significant part of why VPATs exist in the first place, not simply a pass-or-fail certificate of conformance.
However, these noted exceptions and violations should preferably be accompanied by a pledge or estimated time frame for addressing the issue.
Firms may be reluctant to reveal such strategic information to the public, and for good reason, but it would be very good for clients/vendors to know that a product is due to have (say) its keyboard operation fully working by Q4 of next year.

What I'm wondering about is whether there is a way for OPAT to formally indicate a vendor pledge, or timeframe estimate, without this necessarily being a fixed legal commitment. Strategies and priorities change, so it would be good if this could be included, with appropriate legal caveats (expressing levels of confidence from "unknown", through "best guess" to "certainty"). 

I wonder if this legal groundwork were in place, vendors would be more prepared to be transparent and accurate in an OPAT

**Comment by mgifford** [comment]
The violations will always be fixed in the next release, right? :)

Even describing the efforts to resolve the issue might be useful. 

Part of the challenge with the VPAT is that it was limited to the conventions of a Word Document or PDF. Organizing the information needed to be simplified and static. 

An HTML based ACR could concievably include a range of optional elements which would allow a vendor to outline elements like a confidence level. This could potentially be displayed as additional information associated with each SC. 

For an Internal ACR, this is something that could be easily added to help build organizational confidence in delivery. I do think that there would be pushback from vendors on releasing this, and procurement officers would need more training to help them read the findings. 

We are hopign that the framework and tools which we are building here will be flexible enough to be adapted to allow the type of innovative work you are doing @brennanyoung at the same time allowing for the standardization to ensure that organizations can make informed decisions based on comparisons of their ICT.

**Comment by mgifford** [comment]
Would be nice to be able to see where these accessibility issues fall in a product roadmap. Not all software has this though, but for those that do it is a good way to ensure it isn't forgotten (again).
